helpers/formValidation.tsx

```diff
@@ -1,107 +1,121 @@
 import * as Yup from 'yup'
 
-const TEXT_MIN = 2
-const TEXT_MAX = 64
-const PASSWORD_MIN = 6
+enum Constants {
```

Comment:
Can we choose a name for this Enum that contains more meaning? Also, it seems we have a mix of 2 types of constants in this enum: error messages and requirements. IMO we can increase organization and readability of code by separating this into 2 enums

```
enum ErrorMessages {
  LeadingTrailing = 'Leading and trailing value must be alphanumeric character',
  NumbersOnly = 'Numbers only',
  PositiveNumbers = 'Positive numbers only',
  AlphaNumChars = 'Must be alphanumerics characters',
  MustBeLowercase = 'Must be a lowercase string',
  Required = 'Required'
}

enum Requirements {
  TextMin = 2,
  TextMax = 64,
  PasswordMin = 6
}
```